packages/web/src/components/artist/ArtistCover.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+import { ReactComponent as BadgeArtist } from 'assets/img/badgeArtist.svg'
+import { ID } from 'common/models/Identifiers'
+import {
+  CoverPhotoSizes,
+  ProfilePictureSizes,
+  SquareSizes,
+  WidthSizes
+} from 'common/models/ImageSizes'
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import FollowsYouBadge from 'components/user-badges/FollowsYouBadge'
+import UserBadges from 'components/user-badges/UserBadges'
+import { useUserCoverPhoto } from 'hooks/useUserCoverPhoto'
+import { useUserProfilePicture } from 'hooks/useUserProfilePicture'
+
+import styles from './ArtistCard.module.css'
+
+const gradient = `linear-gradient(180deg, rgba(0, 0, 0, 0.001) 0%, rgba(0, 0, 0, 0.005) 67.71%, rgba(0, 0, 0, 0.15) 79.17%, rgba(0, 0, 0, 0.25) 100%)`
+
+type ArtistCoverProps = {
+  userId: ID
+  name: string
+  handle: string
+  isArtist: boolean
+  doesFollowCurrentUser: boolean
+  onNameClick: () => void
+  coverPhotoSizes: CoverPhotoSizes
+  profilePictureSizes: ProfilePictureSizes
+}
+
+export const ArtistCover = ({
```

Comment:
nit: Rename this to `ArtistCardCover` so it's clearer what it's used for/how it's meant to be used?